docs/HowToUseDocker.md

```diff
@@ -0,0 +1,51 @@
+# How to build with Docker
+
+If docker client is installed, you can build TizenRT without installing extra libaries such as toolcahin, kconfig-frontend, etc.
+With docker, the build process would be exactly same with ordinary one.
+
+## Getting the Docker
+See https://docs.docker.com/install/linux/docker-ce/ubuntu/ to install docker on the system.
+
+### Getting the sources
+
+```bash
+git clone https://github.com/Samsung/TizenRT.git
+cd TizenRT
+TIZENRT_BASEDIR="$PWD"
+```
+
+### How to Build
+
+Configure the build from *$TIZENRT_BASEDIR/os/tools* directory
+```bash
+cd os/tools
+./configure.sh <board>/<configuration_set>
+```
+The configuration file is named *defconfig*,  
+and resides under the relative path \<board\>/\<configuration_set\> rooted at *build/configs*.  
+To check the different \<board\>/\<configuration_set\> combinations supported, type below:
+```bash
+./configure.sh --help
+```
+
+After configuring above, configuration can be modified through *make menuconfig* from *$TIZENRT_BASEDIR/os*.
+```bash
+cd ..
+./dbuild.sh menuconfig
```

Comment:
I got a failure without 'sudo'.

Reply:
@davidfather Would you like to refer this page? https://docs.docker.com/install/linux/linux-postinstall/

Reply:
I got it.  But still need to tell developers about how to run docker without sudo
by noting the link @gcjjyy mentioned.